Prod-passenger.Dockerfile

```diff
@@ -0,0 +1,60 @@
+# Ruby version 2.7.4
+FROM phusion/passenger-ruby27:2.0.1
+
+SHELL ["/bin/bash", "-c"]
+
+# Set correct environment variables.
+ENV HOME /root
+
+# Use baseimage-docker's init process.
+CMD ["/sbin/my_init"]
+
+# Enable nginx
+RUN rm -f /etc/service/nginx/down
+
+# Remove default nginx site
+RUN rm /etc/nginx/sites-enabled/default
+
+# Add nginx site for simple-server
+ADD webapp.conf /etc/nginx/sites-enabled/webapp.conf
+
+# Default directory setup
+ENV INSTALL_PATH /home/app
+RUN mkdir -p $INSTALL_PATH
+WORKDIR $INSTALL_PATH
+
+# Create cron logs directory
+RUN mkdir -p /home/deploy/apps/simple-server/shared/log
+
+# Copy application files
+COPY --chown=app:app ./ $INSTALL_PATH
+
+## Install dependencies
+# Yarn
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg -o /root/yarn-pubkey.gpg && apt-key add /root/yarn-pubkey.gpg
+RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" > /etc/apt/sources.list.d/yarn.list
+RUN apt-get update && apt-get install -y --no-install-recommends yarn
+# Redis and Postgres
+RUN apt-get update && apt-get install -y redis-server postgresql-client jq
+# Node
+RUN curl -sL https://deb.nodesource.com/setup_16.x | bash -
+RUN apt-get update && apt-get install -y --no-install-recommends nodejs
+# Install cron
+RUN apt-get install -y cron
+# Install vim
+RUN apt-get install -y vim
+
+# Configure rails env
+ENV RAILS_ENV production
+ENV RAILS_SERVE_STATIC_FILES true
+ENV RAILS_LOG_TO_STDOUT true
+ENV BUNDLE_VERSION 2.2.29
```

Comment:
We use 2.3.22 now. This will also bump up frequently, I wonder if there's a way we can specify this in the application code and have ruby be smart enough to respect it.

Reply:
Do we specify the bundler version somewhere in the application code?

Reply:
Now we are fetching the version from gem file
https://bundler.io/blog/2019/05/14/solutions-for-cant-find-gem-bundler-with-executable-bundle.html